src/platform/Linux/bluez/Helper.cpp

```diff
@@ -466,6 +461,9 @@ static gboolean BluezCharacteristicAcquireWrite(BluezGattCharacteristic1 * aChar
         goto exit;
     }
 
+    // Stack-allocated dict still needs to be cleared after initialization
```

Comment:
Should this also be called in the `else` block above?

Reply:
Many thanks for the review! I've messed that up ;p